### PR TITLE
Document composer sync recipe force option

### PR DESCRIPTION
**Pull request by noniagriconomie**
Small improvement on the upgrade packages process
It may help the process of upgrading/reinstalling to get latest recipe code

Also fix https://github.com/symfony/symfony-docs/issues/12698 👍 

edit: maybe check also https://github.com/symfony/symfony-docs/pull/13320 as it is very similar :)

### CODE REVIEW

File: setup/_update_all_packages.rst.inc [hunk 1]
```diff
@@ -17,3 +17,16 @@ this safely by running:
     breaking changes.
 
 .. _`version constraints`: https://getcomposer.org/doc/articles/versions.md
+
+If you are in a default Symfony application that uses :ref:`Symfony Flex <symfony-flex>`
+and Symfony Recipes, you may also want to reinstall them to their latest version:
+
+.. code-block:: terminal
+
+    $ composer sync-recipes --force
```
Comment (maxhelias) [inline setup/_update_all_packages.rst.inc:26]: We should recommend this command `composer recipes:install vendor/package --force -v` to update one by one and `-v` for more information about the update

Comment (noniagriconomie) [reply setup/_update_all_packages.rst.inc:26]: @maxhelias yes i see your idea
but as the upper code is a full `composer update`, that is why i added here a full "recipes update"

the -v option can be added indeed :)

Comment (maxhelias) [reply setup/_update_all_packages.rst.inc:26]: So just `composer recipes:install --force -v` will have the same behavior

**Comment by wouterj** [comment]
Hi @noniagriconomie! I'm a bit unsure about how I missed this PR when merging #13320 . To me, it seems like the changes in this PR would add duplicate information the upgrade guides now that the other PR is merged. Is there anything left to do in this PR, or should it be closed?

Again, sorry for loosing your changes and thanks a lot for the time you've spent to improve the docs for everyone!

**Comment by noniagriconomie** [comment]
@wouterj no worry, the main goal was to properly document the process of updating the recipes
this one (`sync-recipe`) or the other (`install --force`) it is ok for me

just to be precise, i just think in a doc part of "Upgrading/Updating Recipes"
i found the "install" weird, thus prefering the "sync" wording/action

Feel free to close this one if the wording is ok
Also i think you can close the related issue https://github.com/symfony/symfony-docs/issues/12698 :)

**Comment by javiereguiluz** [comment]
I agree with Wouter and that's why we're closing this without merging. Please, note that this RST file is included just before the other RST that explains the update of recipes (https://github.com/symfony/symfony-docs/blob/5.x/setup/_update_recipes.rst.inc) and which talks about `--force` option and the problem that it introduces.

So, this RST file is included in other articles which don't include the "_update_recipes.rst.inc", but that's fine because it's about updating patch/minor Symfony versions.

Thanks for understanding!

**Pull request closed without merging**